go.sum

```diff
@@ -0,0 +1,26 @@
+github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ=
```

Comment:
go.sum shouldn't be committed. Should be in the .gitignore file.

Reply:
hmmm... not sure how that go there.  fixing.